new-frontiers-14/frontier-station-14
# Smooth transition for walls and windows icons

## About the PR
Commented out some lines I know nothing about to restore smooth transitions between walls/windows/secret doors icons

## Why / Balance
Visual QoL

## How to test
Place different walls and windows, observe smooth transition from one entity to another

## Media
![2024-7-04_04 08 14](https://github.com/new-frontiers-14/frontier-station-14/assets/65374927/d2aaf521-6f97-4b09-9347-76323a12ffc5)
![2024-7-04_04 08 32](https://github.com/new-frontiers-14/frontier-station-14/assets/65374927/0445972a-e25d-4ebc-8f91-bfea7e882cf2)
![2024-7-04_04 09 45](https://github.com/new-frontiers-14/frontier-station-14/assets/65374927/1e5d78b4-8804-4e12-9f01-e0c9ae44422d)
![2024-7-04_04 10 49](https://github.com/new-frontiers-14/frontier-station-14/assets/65374927/339de55b-6c12-4661-aed9-dce8d75048c7)
![2024-7-04_04 13 11](https://github.com/new-frontiers-14/frontier-station-14/assets/65374927/201fd0cc-9d02-44f6-9552-d512c022f89b)
![2024-7-04_04 15 04](https://github.com/new-frontiers-14/frontier-station-14/assets/65374927/8976c715-c3a6-44fe-9607-dbf522f34ed7)
![2024-7-04_04 18 01](https://github.com/new-frontiers-14/frontier-station-14/assets/65374927/5168496d-3672-47c9-b587-e158341d2f81)

## Breaking changes
changes to base game files

**Changelog**
:cl: whatstone and whatstone
- tweak: Walls/windows/secret doors icons once again smoothly transition into each other.
